src/components/FullHeader/TrialButton.js

```diff
@@ -59,7 +70,15 @@ const TrialButton = props => {
 		return buttonWithMessage({
 			daysLeft: daysLeftValue,
 			trialMessage: tooltipTitle,
-			redirectURL: `/?id=${cluster}&subscription=true`,
+			// Given scenario of type -> cluster, this will require iframe url with arc metadata to be supplied
+			redirectURL: {
+				pathname: `/clusters/${cluster}/explore`,
+				state: {
+					arc: deployment ? getAddon('arc', deployment) : {},
```

Comment:
We might already be having getAddon as util in the cluster or you can just pass this value from the parent component